src/components/02_atoms/MultipleFields/MultipleFields.js

```diff
@@ -172,11 +171,12 @@ class MultipleFields extends Component {
       state: { newItemAdded },
       props: { label, value: values, component, onChange },
     } = this;
+    const usedValues = values.length ? values : [this.createEmptyItem()];
```

Comment:
Here we construct an empty item. I think that's fine, given that its mostly a UI fact that there should be an empty element by default